core/blockchain.go

```diff
@@ -2665,6 +2669,17 @@ func (bc *BlockChain) SubscribeBlockProcessingEvent(ch chan<- bool) event.Subscr
 	return bc.scope.Track(bc.blockProcFeed.Subscribe(ch))
 }
 
+// Quorum
 func (bc *BlockChain) SupportsMultitenancy(context.Context) (*proto.PreAuthenticatedAuthenticationToken, bool) {
 	return nil, bc.isMultitenant
 }
+
+// Quorum
+func (bc *BlockChain) SaveRevertReason(enabled bool) {
```

Comment:
I would follow the [effective Go for getters/setters](https://golang.org/doc/effective_go#Getters).
```suggestion
func (bc *BlockChain) SetSaveRevertReason(enabled bool) {
```